### ISSUE TITLE
warning: User instrumentation from opcache: opcache status information is not an array

### ISSUE THREAD

**Issue by csandanov** [newrelic/newrelic-php-agent#594]
## Description

Running CLI commands when newrelic and opcache enabled (but disabled for CLI `opcache.enable_cli=0`) produces the warning:
```
warning: User instrumentation from opcache: opcache status information is not an array
```

Most likely it's not checked if opcache enabled for CLI when executing `opcache_get_status` https://github.com/newrelic/newrelic-php-agent/blob/main/agent/php_execute.c#L1681

## Steps to Reproduce

I reproduced the issue while running `drush` in Drupal 9 on PHP 8.1

- Run vanilla Drupal 9 docker container with enabled newrelic and opcache:
```
docker run --rm -e PHP_NEWRELIC_LICENSE=[license]-e PHP_NEWRELIC_ENABLED=1 -e PHP_OPCACHE_ENABLED=1 wodby/drupal:9
```
- Get docker container id by running `docker ps` and then access it via `docker exec -ti [container-id] sh`
- Wait a bit for a newrelic daemon to fully startup and connexct
- Run `drush version` and you will see the warning:
```
$ drush version
2022-12-22 12:16:19.794 +0000 (163 163) warning: User instrumentation from opcache: opcache status information is not an array
Drush version : 11.4.0
```

If you enable opcache for CLI as:
```
docker run --rm -e PHP_NEWRELIC_LICENSE=[license]-e PHP_NEWRELIC_ENABLED=1 -e PHP_OPCACHE_ENABLED=1 PHP_OPCACHE_ENABLE_CLI=1 wodby/drupal:9
```
You will get a different warning:
```
$ drush version
2022-12-22 12:21:24.252 +0000 (167 167) warning: User instrumentation from opcache: missing 'scripts' key in status information
Drush version : 11.4.0
```

## Expected Behavior

No warning should be produced

**Comment by bduranleau-nr** [comment]
Hi @csandanov - Thanks for bringing this to our attention.
We took a look at these messages and determined that the PHP 8.1 code path requires more strict checks before opcache status is queried. Additionally the severity aligns more with `debug` than `warning` - expect a fix for this in a future release.

We have a few follow up questions if you don't mind:
- What version of the agent are you using?
- Are you seeing any impact to performance?
- Is instrumentation working as expected?

**Comment by csandanov** [comment]
Newrelic version is `10.3.0.315-4571e4f3699a`. It's hard to tell about the performance impact but I think there's nothing noticeable. Instrumentation seems to be working fine

**Comment by misiekn** [comment]
Fixed in [10.5.0 release](https://github.com/newrelic/newrelic-php-agent/releases/tag/v10.5.0.317).
